src/Data/Vec/Recursive.agda

```diff
@@ -148,3 +155,18 @@ zip = zipWith _,_
 
 unzip : ∀ n → (A × B) ^ n → A ^ n × B ^ n
 unzip = unzipWith id
+
+private module ↔ {ℓ} = IsEquivalence (↔-isEquivalence {ℓ})
```

Comment:
Hmm having to reparameterise the module every time we want to use reflexivity/transitivity of inverse is a bit ugly. Instead could we add proofs of `↔-refl`, `↔-trans` etc. directly to `Function.Properties.Inverse` and then just use them directly?

Reply:
I put this module in `Function.Properties.Inverse`.